packages/core/http/core-http-server/src/router/route.ts

```diff
@@ -159,6 +159,9 @@ export interface RouteConfigOptions<Method extends RouteMethod> {
      */
     idleSocket?: number;
   };
+
+  /** Human-friendly description of this endpoint */
+  description?: string;
```

Comment:
Can remove this and introduce later, just thought it doesn't hurt to include in this PR.